analysis/analyzer.go

```diff
@@ -91,7 +88,7 @@ var defaultIgnoreDirs = []string{
 	".vitepress",
 }
 
-func RunAnalyzers(path string, analyzers []*Analyzer) ([]*Issue, error) {
+func RunAnalyzers(path string, analyzers []*Analyzer, fileFilter func(string) bool) ([]*Issue, error) {
```

Comment:
Assuming the Analyzer refers to a checker here, can we rename this to RunCheckers?